components/common/Card.tsx

```diff
@@ -3,22 +3,30 @@ import { Card as BSCard, CardBody } from 'reactstrap';
 import styled from 'styled-components';
 import { transparentize } from 'polished';
 
-const StyledCard = styled(BSCard)`
+const StyledCard = styled(BSCard)<{
+  $customColor?: string;
+  $customBackgroundColor?: string;
+}>`
   width: 100%;
   transition: all 0.5s ease;
   overflow: hidden;
   border-width: ${(props) => props.theme.cardBorderWidth};
   border-radius: ${(props) => props.theme.cardBorderRadius};
   background-color: ${(props) =>
-    props.customcolor ? props.customcolor : props.theme.themeColors.white};
+    props.$customBackgroundColor
+      ? props.$customBackgroundColor
+      : props.theme.themeColors.white};
```

Comment:
Little nitpick for the ternary statements in this component:

```suggestion
    props.$customBackgroundColor || props.theme.themeColors.white};
```